README.md

```diff
@@ -20,14 +20,38 @@ The API uses [Django REST framework](http://www.django-rest-framework.org/)
 
 
 ## Installation - Git and repository setup
-
+### Setting up your Git user and cloning the repo
 ```bash
 $ git config --global core.autocrlf false
 $ git config --global user.name "<your github username>"
 $ git config --global user.email your.github@email.com
 $ git clone git@github.com:dotkom/onlineweb4.git
 $ cd onlineweb4
 ```
+### Setting up the onlineweb4
+Download dependencies by running
+```bash
+make
+```
+
+Lets apply all the django changes to the database by running
```

Comment:
Should be Let's as in "Let us" , rather than Lets as in "the cop lets you go". Alternatively change it to "Let us"
```suggestion
Let's apply all the django changes to the database by running
```